docs/acme-divergences.md

```diff
@@ -0,0 +1,82 @@
+# Boulder divergences from ACME
+
+While Boulder attempts to implement the ACME specification as strictly as possible
+there are places at which we will diverge from the letter of the specification for
+various reason.
+
+This document details these differences, since ACME is not yet finalized it will be
+updated to be up to date with the most recent numbered draft that has been published.
+
+Current draft: [`draft-ietf-acme-acme-03`](https://tools.ietf.org/html/draft-ietf-acme-acme-03).
+
+## [Section 5.5.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-5.5)
+
+Boulder does not provide a `Retry-After` header when a user hits a rate-limit, nor
+does it provide `Link` headers to further documentation on rate-limiting.
+
+## [Section 5.6.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-5.6)
+
+Boulder doesn't return errors under the `urn:ietf:params:acme:error:` namespace
+but instead uses the `urn:acme:error:` namespace from [draft-ietf-acme-01 Section 5.4](https://tools.ietf.org/html/draft-ietf-acme-acme-01#section-5.4).
+
+Boulder also uses `invalidEmail` in place of the error `invalidContact` defined in
+ACME and doesn't implement the `caa` and `dnssec` errors.
```

Comment:
nit: I think it makes sense to split this into two sentences. One for `invalidEmail` vs `invalidContact`, one for not implementing `caa` and `dnssec` errors.